parachain-template/polkadot-launch/config.json

```diff
@@ -17,7 +17,7 @@
 	},
 	"parachains": [
 		{
-			"bin": "../target/release/parachain-collator",
+			"bin": "../target/release/polkadot-parachain",
```

Comment:
as this is part of the template, should the bin being executed be `polkadot-template-node` ?